applications/asset_tracker/prj_nrf9160_pca20035ns.conf

```diff
@@ -61,7 +61,11 @@ CONFIG_ADXL362_INACTIVITY_THRESHOLD=200
 CONFIG_ADXL362_INTERRUPT_MODE=1
 CONFIG_TEMP_USE_EXTERNAL=y
 CONFIG_TEMP_DEV_NAME="BME680"
-CONFIG_BME680=y
+
```

Comment:
Config CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y is required when using toolchain zephyr-sdk 0.10.2. Without this no sensor data will be sent to cloud

Reply:
done